community/command-line/src/main/java/org/neo4j/commandline/admin/AdminCommand.java

```diff
@@ -81,9 +81,9 @@ public String name()
         public abstract String summary();
 
         /**
-         * @return AdminCommandSegment the command using the provider is grouped under
+         * @return AdminCommandSection the command using the provider is grouped under
          */
-        public abstract AdminCommandSegment segment();
+        public abstract AdminCommandSection commandSection();
```

Comment:
I initially started with a simple `String` implementation. But the idea of sections added a lot of behaviour to the simple `String` object for instance: Having to sort and map the `String` to different `AdminCommand.Provider` which makes me think that there's more to the sections than it being a simple `String`